source/our-stacks/ios/index.md

```diff
@@ -3,4 +3,67 @@ title: iOS
 date: 2016-09-29 15:27:11
 layout: stack
 ---
-Total iOS
\ No newline at end of file
+
+# iOS
+
+iOS developers at Nodes build user-centric applications and extensions which are distributed on the App Store or through enterprise distribution platforms.
+
+We deliver a range of applications for different platforms and devices like the iPhone, iPad, Apple Watch or Apple TV. 
+
+### Tasks
+ - Writing business and view logic with testability in mind
+ - Setting up and maintaining unit and UI tests
+ - Integrating with in-house or external APIs and services
+ - Maintaining and setting up Apple services and applications 
+ - Consulting with UX and clients to provide best possible product
```

Comment:
Maybe change 'UX' with 'our UX team'. For someone outside Nodes, this may be confusing